src/cryptsetup/cryptsetup.c

```diff
@@ -747,11 +747,13 @@ static int attach_luks_or_plain_or_bitlk_by_fido2(
 
         if (arg_fido2_cid) {
                 if (!key_file && !key_data)
-                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "FIDO2 mode selected but no key file specified, refusing.");
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "FIDO2 CID mode selected but no keyfile specified, refusing.");
 
                 rp_id = arg_fido2_rp_id;
                 cid = arg_fido2_cid;
                 cid_size = arg_fido2_cid_size;
+
+                required = FIDO2ENROLL_PIN; /* For backwards compatibility, PIN is required by default. */
```

Comment:
I set UP as well in my PR: https://github.com/systemd/systemd/pull/19558 I think it would have been set too?

Reply:
In the status quo ante we turned off "uv" and left "up" and "clientPin" in its default values, i.e. to whatever libfido2 set it to, which according to the docs is  FIDO_OPT_OMIT, which leaves it to the authenticator...

Reply:
as far as I could see, with yubikeys (which are the most popular ones I think) it meant both up and pin were enabled by default